### ISSUE TITLE
Improve understanding of properties of determinants

### ISSUE THREAD

**Issue by nbro** [pennal/LinAlgNotes#10]
In chapter 6, starting from page 45, we have a simple enumeration of the properties of the determinants and after that we have the examples and/or proofs of the properties.

From my point of view, it would be more understandable if we add the examples and/or proofs of each property immediately after the definition of the property, this because one cannot memorize immediately a bunch of properties and then remember on the fly also the order in which they were presented when reading the examples/proofs below. 

Basically, when reading the examples of each property, we need always to go up again to revise the properties and check to which property is this example/proof related...

Anyway, this was just my first impression when reading this part...
